src/main/java/com/hades/user/UserController.java

```diff
@@ -26,7 +25,7 @@
 
 	@PermitEndpoint
 	@RequestMapping(path = "/register", method = POST)
-	public ResponseEntity<LoginInfoDTO> register(@Valid @RequestBody UserDTO userDTO) {
```

Comment:
Como comentado no outro pr: Parace que precisa mesmo do `@RequestBody`, sem ele estou levando um `400` no angular com a mensagem `Validation failed for object='loginInfo'. Error count: 2`